stanford/cme/db.yml

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ansible-playbook
```

Comment:
I need to run this for stage since it doesn't have the subnets/SG. How would I do it without at least the stage play? Actually, I think I need 2 stage plays; 1 and 2.

Reply:
@dcadams That's the thing; you don't need to merge and run this before doing the other work. Sure, the SG/subnet needs to exist before you technically create the instance itself, but it doesn't make sense to do this in 2 discrete phases. We don't need to, nor should we, run these playbooks before working on the plays that run the `rds` module.

For all of the other clusters, we did everything in a single PR, eg: [3]. We didn't first PR to create the `analytics-jenkins` SG/subnet and only then later create another PR to actually provision the `analytics-jenkins` cluster. We did it all at once. 

The biggest reason for this is that changes you make in the 2nd "phase" may alter assumptions/decisions from the 1st. Like here, we have a stage play called `db`. But when you actually go to provision the analytics-specific database, we won't be calling that play `db`; we would give it something more specific like `analyticsdb` or some such.

The current approach puts the cart before the horse.

On the ES PR, it made sense to make one PR that just creates the SG/subnet, but that's because there wasn't an Ansible phase 2; we created the ES host manually, since Ansible couldn't. Here for RDS, we can create both the subnet group [1] and the instance [2] directly via ansible.

I can't see a compelling reason to perform these changes in 2 phases, when we're just going to have to rename most of these files in the second phase anyway.

[1] https://docs.ansible.com/ansible/latest/modules/rds_subnet_group_module.html
[2] https://docs.ansible.com/ansible/latest/modules/rds_module.html
[3] https://github.com/Stanford-Online/configuration/pull/238

Reply:
It's also just harder to review with the changes split up like this.

I would suggest just moving the SG/subnet bit over to the other PR and close this.